.taskcluster.yml

```diff
@@ -606,7 +537,6 @@ tasks:
                 taskId: {$eval: as_slugid("release")}
                 dependencies:
                   - {$eval: as_slugid("backend_build")}
-                  - {$eval: as_slugid("bot_build")}
```

Comment:
This is a bit of a drawback, but I don't see any solution other than moving everything to Firefox-CI.

Reply:
That's the long term goal, but that won't happen soon, as the images built with the old DinD are not compatible with Heroku